### DISCUSSION TITLE
Added: LDM Notes for February 14th, 2022

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#5784] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-02-14.md

## Agenda

1. Definite assignment in structs
2. Checked operators

**Comment by MgSam** [comment]
> While there are currently no members of the LDT who were on the design team at that time, we have a decent understanding of the reasoning here...

> We will follow up with emeritus members of the LDT who were present when the original restrictions were made, but barring any large revelations we like this approach.

I know Anders isn't involved in the day-to-day of C# anymore but I'm interested in who else was part of the original design team and is still at Microsoft?

**Reply by 333fred** [reply]
I don't generally include names in the notes, sorry 🙂.
